packages/@okta/vuepress-site/docs/guides/terraform-configure-device-login-standards/main/index.md

```diff
@@ -0,0 +1,377 @@
+Require users to authenticate only with phones and computers that meet your organization’s requirements.
+
+---
+
+#### Learning outcomes
+
+* Improve device security, resource management, and consistency with Okta device assurance policies.
+
+* Review device compliance in the Okta Admin Console.
+
+* Create Okta device assurance policies in Terraform.
+
+* Enforce device requirements during Okta user authentication.
+
+#### What you need
+
+* Familiarity with Terraform terms: configuration, resources, state, and commands. See the [Terraform overview](/docs/guides/terraform-overview).
+
+* An Okta organization.
+
+* A [Terraform configuration](/docs/guides/terraform-enable-org-access/main) that can access your Okta org.
+
+* An Okta pricing plan that [enables Device Assurance in your organization](#confirm-your-org-supports-device-assurance).
+
+* An Okta user account with the Super Administrator role.
+
+* A list of your organization’s requirements for user devices.
+
+## Overview
+
+Using a device's attributes to control access to your organization requires configuring two sets of policies. Device assurance policies configure device requirements, such as a minimum OS version. Authentication policies include rules that use the assurance policies to authorize a connection.
+
+You can manage both types of policies with Terraform.
+
+The benefits of device assurance policies include:
+
+* Improve security by checking that devices meet minimum standards, such as the latest OS updates.
+
+* Improve device configuration consistency, which reduces support burden and vulnerability risks.
+
+* Simplify resource management, policy adjustments, security audits, and policy enforcement by displaying device compliance on one dashboard.
+
+Okta supports device assurance for Android, iOS, macOS, Windows, and ChromeOS platforms. Each platform requires a minimum OS to support the device assurance feature. For more information, see the [minimum supported OS versions](https://help.okta.com/oie/en-us/content/topics/identity-engine/devices/device-assurance.htm).
+
+For user instructions on how to check device compliance, see the [main Okta article for device assurance](https://help.okta.com/oie/en-us/content/topics/identity-engine/devices/device-assurance.htm).
+
+## Confirm your org supports Device Assurance
+
+Some of the steps in this article require that your Okta org and pricing plan support Device Assurance.
+
+1. Open your Okta organization **Admin Dashboard**.
+
+2. In the left navigation, select **Security** and then **Authentication Policies**.
+
+Note: Ignore the **Security > Device Assurance Policies** navigation item for this test.
+
+3. Select a policy, whether it's default or one you’ve already created.
+
+4. Select **Add a policy**.
+
+5. Search for a field with the content:
+
+      ```
+      AND Device assurance policy is
+      ```
+
+6. If you found that field then you can create authentication policies that reference device assurance policies.
```

Comment:
I modified this further:

> If the field exists then you can create device assurance policies and reference them in authentication policies. Otherwise, contact Okta Support about feature availability.